### ISSUE TITLE
@eles macro

### ISSUE THREAD

**Issue by mattsignorelli** [bmad-sim/AcceleratorLattice.jl#64]
It would be nice to have another macro `@eles`  which takes in a block and processes each element so instead of 

```julia
@ele qf = Quadrupole(k1=0.36,L=0.5)
@ele qd = Quadrupole(k1=-0.36,L=0.5)
...
```

Once could also do

```julia
@eles begin
qf = Quadrupole(k1=0.36,L=0.5)
qd = Quadrupole(k1=-0.36,L=0.5)
...
end
```
[This discussion shows exactly how one could do this](https://discourse.julialang.org/t/use-multi-line-arguments-to-macros-without-commas-with-a-new-block-structure/20577/5)

**Comment by DavidSagan** [comment]
Very nice. It is on my list...

**Comment by mattsignorelli** [comment]
implemented in #71
